engines/stark/ui/menu/mainmenu.h

```diff
@@ -55,9 +55,22 @@ class MainMenuScreen : public StaticLocationScreen {
 	void loadHandler();
 	void creditsHandler();
 	void settingsHandler();
+	void boxHandler();
 	void quitHandler();
 };
 
+/**
+ * The version info text
+ */
+class VersionInfoText : public StaticLocationWidget {
+public:
+	VersionInfoText();
+	virtual ~VersionInfoText() {}
+
+private:
+	static const int _copyrightSymbol = -87, _posX = 16, _posY = 419;
```

Comment:
`_copyrightSymbol` is the ISO-8859-1 code for the copyright symbol. It would be easier to understand what it is as a hexadecimal number (0xA9).

Also please don't put the position on the same line, they are really unrelated.